### ISSUE TITLE
Probably docker container error

### ISSUE THREAD

**Issue by aurelhh** [DewGew/Domoticz-Google-Assistant#338]
**Describe the bug:**  
With Docker on Synology since DZGA Version: 1.23.10 no device appears, logs just says : Error in getting devices and setting
settings are ok on the dedicated page.
Its not a 401 error, new user created with 8ch password, etc 'allow basic-auth'  activated. Everything ok on 1.23.7
get error running a bash on the container 

**Logs:**
2023-08-23 20:20:33 - INFO - ========
2023-08-23 20:20:33 - INFO - Started DZGA v1.23.10 server at port 3030
2023-08-23 20:20:33 - INFO -  
2023-08-23 20:20:33 - INFO -    Visit http://localhost:3030/settings to access the user interface
2023-08-23 20:20:33 - INFO -  
2023-08-23 20:20:33 - INFO - =========
2023-08-23 20:20:33 - INFO - Visit the Actions on Google console at http://console.actions.google.com
2023-08-23 20:20:33 - INFO - Under Develop section, replace the fulfillment URL in Actions with:
2023-08-23 20:20:33 - INFO -    https://[your public url]/smarthome
2023-08-23 20:20:33 - INFO -  
2023-08-23 20:20:33 - INFO - In Account linking, set the Authorization URL to:
2023-08-23 20:20:33 - INFO -    https://[your public url]/oauth
2023-08-23 20:20:33 - INFO -  
2023-08-23 20:20:33 - INFO - Then set the Token URL to:
2023-08-23 20:20:33 - INFO -    https://[your public url]/token
2023-08-23 20:20:33 - INFO -  
2023-08-23 20:20:33 - INFO - Finally press 'SAVE' and then 'TEST' button
2023-08-23 20:20:33 - INFO - =======
2023-08-23 20:20:33 - ERROR - Error in  getting devices and settings
```

**Desktop (please complete the following information):**
 - Domoticz version <beta 2023.1 build 15192-->
 - Dzga version <!-- 1.27.10 -->
 - Python Version <!-docker image -->

**Comment by RunnicFusion** [comment]
Experiencing same behavior.

- Upgraden to 1.23.10
- Running DZGA inside docker
- Before everyting was ok
- Right now error "2023-08-24 17:02:40 - ERROR - Error in  getting devices and settings" is thrown


Did reinstall DZGA, created new app in google actions, setup new smart-home-key.json with correct serviceAccount settings and did some testing. Can't get it working.


Full log:
```
2023-08-24 17:06:47 - DEBUG - http://192.168.1.18:8080 "GET /json.htm?type=command¶m=getdevices&plan=0&filter=all&used=true HTTP/1.1" 200 22
2023-08-24 17:06:47 - ERROR - Error in  getting devices and settings
2023-08-24 17:07:07 - DEBUG - Starting new HTTPS connection (1): accounts.google.com:443
2023-08-24 17:07:07 - DEBUG - https://accounts.google.com:443 "POST /o/oauth2/token HTTP/1.1" 200 None
2023-08-24 17:07:07 - DEBUG - Starting new HTTPS connection (1): homegraph.googleapis.com:443
2023-08-24 17:07:07 - DEBUG - https://homegraph.googleapis.com:443 "POST /v1/devices:requestSync HTTP/1.1" 404 None
```

**Comment by DewGew** [comment]
An other user had problems with docker and kubernet. 

Try his https://github.com/DewGew/Domoticz-Google-Assistant/pkgs/container/domoticz-google-assistant

**Comment by aurelhh** [comment]
> An other user had problems with docker and kubernet.
> 
> Try his https://github.com/DewGew/Domoticz-Google-Assistant/pkgs/container/domoticz-google-assistant

Hi and tank you
unfortunately, tried this and didnt work, same trouble

![Capture d’écran du 2023-08-26 22-28-02](https://github.com/DewGew/Domoticz-Google-Assistant/assets/59808314/28355b75-9910-410f-b72e-1cbadeab4186)
![Capture d’écran du 2023-08-26 22-29-01](https://github.com/DewGew/Domoticz-Google-Assistant/assets/59808314/b5555154-d738-410e-89e0-8cf092b498eb)

**Comment by ayasystems** [comment]
I have the same error

`Content-Length: 79
Content-Type: application/json;charset=UTF-8
Google-Assistant-API-Version: v1
Authorization: Bearer 75PXKY19s7qo8vC5muzr
User-Agent: Mozilla/5.0 (compatible; Google-Cloud-Functions/2.1; +http://www.google.com/bot.html)
Accept-Encoding: gzip, deflate, br
{
  "inputs": [
    {
      "intent": "action.devices.SYNC"
    }
  ],
  "requestId": "9467163695332427772"
}
Google Assistant requests an action.devices.SYNC
Starting new HTTP connection (1): 192.168.3.241:80
http://192.168.3.241:80 "GET /json.htm?type=command&param=getdevices&plan=20&filter=all&used=true HTTP/1.1" 200 22
'result'
Error handling message {'inputs': [{'intent': 'action.devices.SYNC'}], 'requestId': '9467163695332427772'}: {'errorCode': 'unknownError'}
10.30.10.200 - - [07/Sep/2023 12:44:30] "POST /smarthome HTTP/1.1" 200 -
https://homegraph.googleapis.com:443 "POST /v1/devices:requestSync HTTP/1.1" 500 None
10.30.10.200 - - [07/Sep/2023 12:44:30] "POST /settings HTTP/1.1" 404 -`

**Comment by ayasystems** [comment]
I get it working finally.... Try to pair again from google home android app

**Comment by aurelhh** [comment]
> I get it working finally.... Try to pair again from google home android app

tk you, but already tried and not working in my case.

**Comment by aurelhh** [comment]
> @DewGew
> 
> Experiencing same behavior.
> 
>     * Upgraden to 1.23.10
> 
>     * Running DZGA inside docker
> 
>     * Before everyting was ok
> 
>     * Right now error "2023-08-24 17:02:40 - ERROR - Error in  getting devices and settings" is thrown
> 
> 
> Did reinstall DZGA, created new app in google actions, setup new smart-home-key.json with correct serviceAccount settings and did some testing. Can't get it working.
> 
> Full log:
> 
> ```
> 2023-08-24 17:06:47 - DEBUG - http://192.168.1.18:8080 "GET /json.htm?type=command¶m=getdevices&plan=0&filter=all&used=true HTTP/1.1" 200 22
> 2023-08-24 17:06:47 - ERROR - Error in  getting devices and settings
> 2023-08-24 17:07:07 - DEBUG - Starting new HTTPS connection (1): accounts.google.com:443
> 2023-08-24 17:07:07 - DEBUG - https://accounts.google.com:443 "POST /o/oauth2/token HTTP/1.1" 200 None
> 2023-08-24 17:07:07 - DEBUG - Starting new HTTPS connection (1): homegraph.googleapis.com:443
> 2023-08-24 17:07:07 - DEBUG - https://homegraph.googleapis.com:443 "POST /v1/devices:requestSync HTTP/1.1" 404 None
> ```
> 
> also; relinking my Google Home app fails. added my findins here #339

Hi, did you find a solution ?

**Comment by aurelhh** [comment]
okay i understand it's because of domoticz 2023.1. I ve just read Runnicfusion posts.

**Comment by aurelhh** [comment]
Perfect now with correct domoticz's version, i'm closing subject

**Comment by RunnicFusion** [comment]
> okay i understand it's because of domoticz 2023.1. I ve just read Runnicfusion posts.

Correct. The latest version of DZGA requires also the latest version of Domoticz to be installed.  Otherwise you will end up with funny errors like above. So for me the fix was installing Domoticz 2023.2 next to my DZGA 1.23.10 as mentioned in issue: https://github.com/DewGew/Domoticz-Google-Assistant/issues/339#issuecomment-1692945573
